content/content-delivery/v1/core-resources/collaborators/add-collaborator.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Add a Collaborator
+---
+
+You can set some of the fields available in the collaborator object, below we only list the properties in the example and the possible required fields.
+
+| Property | Description |
+|---|---|
+| `collaborator` | The [collaborator object](#core-resources/collaborators/the-collaborator-object) |
+| `email` | Email that will be used in the space interface for collaborator **required** |
+| `role` | Role of the collaborator, could be admin, editor or custom roles **required** |
+| `space_id` | Numeric id of the collaborator space **required** |
+| `space_role_id` | Numeric id of the space role connected with collaborators **required** |
+
```

Comment:
@iamfortune I think the parameters here need some clarifications:
- `role` would be the role name if it's admin or editor but if it's custom you need to send the role id and if you have more than one id you need to send `multi`;
- the `allow_multiple_roles_creation` is missing. I see it's set when you add multiple roles from the app;
- the `permissions` property is missing too. That's for the `Allow managing subscription` part;
- `space_role_id`: you need to explain how this should look when you have more than one collaborator too (in that case it's null);
- you also need to add the `space_role_ids` property which is set in case you have multiple roles.

I would double-check these things with @romeucarvalho but I just saw them in the network tab of the dev tools when trying to add/update collaborators from the app. These updates should be reflected also in the endpoint for updating a collaborator. I see they are also in the collaborator objects so you might also want to review the consistency across all the docs.